backend/src/data/seeds/fill-db.ts

```diff
@@ -0,0 +1,11 @@
+import { vendorsSeed } from '../seed-data/vendors-seed';
+import { Knex } from 'knex';
+
+const TableName = {
+  VENDORS: 'vendors',
+};
+
+export async function seed(knex: Knex): Promise<void> {
+  await knex(TableName.VENDORS).del();
+  await knex(TableName.VENDORS).insert(vendorsSeed);
+}
```

Comment:
```suggestion
```
we need to seed it directly in the migrations. let's discuss it on the tomorrow call